app/views/sign_up/completions/show.html.erb

```diff
@@ -24,6 +24,17 @@
         sp: content_tag(:strong, decorated_session.sp_name),
       ) %>
 </p>
+<% if @multiple_factors_enabled == false && IdentityConfig.store.select_multiple_mfa_options == false %>
+  <%= render(AlertComponent.new(type: :warning, class: 'margin-bottom-4')) do %>
+    <%= link_to(
+          t('mfa.second_method_warning.link'),
+          login_two_factor_options_url,
+        ) %>
+    <%= t(
+          'mfa.second_method_warning.text',
+        ) %>
```

Comment:
bet one could put this on one line if we wanted
```suggestion
    <%= t('mfa.second_method_warning.text') %>
```

Reply:
This makes sense. I probably did this while I was frustrated with the linter 😬